### DISCUSSION TITLE
Error: ENOENT: no such file or directory, lstat '/vercel/path0/apps/max/.next/server/app/(protected)/page_client-reference-manifest.js'

### DISCUSSION THREAD

**Discussion by timmeade** [vercel/community#3580] [Bug Report]
### Page to Investigate

https://github.com/orgs/vercel/discussions/1209

### Steps to Reproduce

Starting a new issue as followup to https://github.com/orgs/vercel/discussions/1209

Full build with turbo repo of single app.

Error: ENOENT: no such file or directory, lstat '/vercel/path0/apps/max/.next/server/app/(protected)/page_client-reference-manifest.js'

Shows up at the very end.  After good build.  I have tried the items in the other issue page and nothing gets past this.  We cannot deploy.

Ideas?

**Comment by timmeade** [comment]
Here is part of the logs:  As you can the error happens after the site has been fully built.   I hope someone at vercel sees this. i need to deploy this within the next 24 hours.



```
--
09:42:38.972 | max:build: Route (app)                                Size     First Load JS
09:42:38.972 | max:build: ┌ λ /                                      142 B          80.5 kB
09:42:38.972 | max:build: ├ λ /                                      142 B          80.5 kB
09:42:38.972 | max:build: ├ λ /api/auth/[...nextauth]                0 B                0 B
09:42:38.972 | max:build: ├ ○ /auth                                  954 B          91.8 kB
09:42:38.972 | max:build: ├ ○ /auth/signout                          141 B          80.5 kB
09:42:38.972 | max:build: ├ λ /construction                          442 B          91.3 kB
09:42:38.972 | max:build: ├ λ /customer/[customerId]                 49 kB           151 kB
09:42:38.972 | max:build: ├ λ /dashboard                             142 B          80.5 kB
09:42:38.972 | max:build: └ λ /testing                               1.25 kB        85.4 kB
09:42:38.972 | max:build: + First Load JS shared by all              80.4 kB
09:42:38.972 | max:build:   ├ chunks/846-bbe94304916a480d.js         25.7 kB
09:42:38.972 | max:build:   ├ chunks/941d0c15-012502682be5c141.js    52.7 kB
09:42:38.972 | max:build:   ├ chunks/main-app-879a39effee9d304.js    217 B
09:42:38.972 | max:build:   └ chunks/webpack-c0c77798b2ac525d.js     1.74 kB
09:42:38.973 | max:build:
09:42:38.973 | max:build: Route (pages)                              Size     First Load JS
09:42:38.973 | max:build: ─ ○ /404                                   183 B          75.6 kB
09:42:38.973 | max:build: + First Load JS shared by all              75.4 kB
09:42:38.973 | max:build:   ├ chunks/framework-eb124dc7acb3bb04.js   45 kB
09:42:38.973 | max:build:   ├ chunks/main-c627e3cfc39f5a24.js        28.4 kB
09:42:38.973 | max:build:   ├ chunks/pages/_app-9a7e9b0eeefc89ea.js  195 B
09:42:38.973 | max:build:   └ chunks/webpack-c0c77798b2ac525d.js     1.74 kB
09:42:38.973 | max:build:
09:42:38.973 | max:build: λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
09:42:38.973 | max:build: ○  (Static)  automatically rendered as static HTML (uses no initial props)
09:42:38.973 | max:build:
09:42:39.163 |  
09:42:39.163 | Tasks:    1 successful, 1 total
09:42:39.163 | Cached:    0 cached, 1 total
09:42:39.163 | Time:    51.597s
09:42:39.163 | Summary:    /vercel/path0/.turbo/runs/2Tt1khGCeR7yJYsMi0x4RB2ypBl.json
09:42:39.163 |  
09:42:42.702 | Traced Next.js server files in: 3.088s
09:42:45.855 | Error: ENOENT: no such file or directory, lstat '/vercel/path0/apps/max/.next/server/app/(protected)/page_client-reference-manifest.js'

```

**Reply by ChungLeba** [reply]
![image](https://github.com/vercel/community/assets/96033389/517f1de1-30c9-4846-97cd-343c0f0ce238)
Im too

**Reply by ChungLeba** [reply]
oh, i can fix it, it found page.tsx in (protected), please delete it and commit to build

**Reply by sudhagarmsd5** [reply]
Thankyou @ChungLeba 
i too faced this error ENOENT: no such file or directory, lstat '/vercel/path0/.next/server/app/(pages)/page_client-reference-manifest.js'

and i fixed it by deleting the page.tsx file inside the (pages) folder

because we are using the folder name with () enclose brackets to tell the app router not to consider this foldername as a routepath 
and the mistake here is mentioning the file name of page.tsx

:)

**Comment by ChungLeba** [comment]
![image](https://github.com/vercel/community/assets/96033389/25722c17-3cdb-4589-b80d-b3e9c3e5b1a4)

**Comment by khalidkhankakar** [comment]
I am also facing same issue how to fix that

**Reply by shishirkj** [reply]
did u get how to fix this issue

**Comment by erhanyasar** [comment]
I found out a case why it happens (Next.js version: 14.2.2).

Creating a page under a group route causing this error to be thrown.

To say, creating a `page.tsx` under a directory named `(website)` will throw this error.

**Comment by trtin** [comment]
I fixed it by deleting the duplicated page. There is one in the directory and root.

**Reply by PrashantIndurkar** [reply]
Yeah thanks

**Comment by damianlluch** [comment]
I'm getting this same error with NextJS 14
